### PR TITLE
delete default price deltas

### PR DESCRIPTION
**Pull request by toninorair**
Delete default price deltas

### CODE REVIEW

File: poster/src/index.ts [hunk 1]
```diff
@@ -49,10 +36,10 @@ async function run() {
   const price_deltas = JSON.parse(parsed['price-deltas']);
   const assets = <string[]>parsed['asset'];
 
-  // check that price deltas are set up for all assets, otherwise set default delta value
+  // check that price deltas are set up for all assets
   assets.forEach(asset => {
     if (price_deltas[asset] == undefined) {
-      price_deltas[asset] = defaultDeltas[asset];
+      throw new TypeError(`For each asset price delta should be provided, ${asset} asset is not properly configured`)
```
Comment (hayesgm) [inline poster/src/index.ts:42]: Only question is: maybe we just want to have a sane default to make life easy, like 5%? Not that we're using it, but it's easier than getting yelled at. Thoughts?

**Review by hayesgm — APPROVED** [review]
Looks great- one comment about maybe having _some_ default

**Review by zack-eth — APPROVED** [review]
LGTM, here is the corresponding PR: https://github.com/compound-finance/open-price-feed-poster/pull/12

File: poster/src/index.ts (hunk 1 above)
Comment (antonina-cherednichenko) [reply poster/src/index.ts:42]: that's what we had before, but then I switched it to different deltas. I would say let's just set all deltas in the poster, it's clear approach and it forces us to properly configure poster and really understand deltas

Comment (antonina-cherednichenko) [reply poster/src/index.ts:42]: kind of 'fail early, fail hard' if you forget about an asset

Comment (hayesgm) [reply poster/src/index.ts:42]: 👍 Sounds good. I like this more now come to think of it. Explicit rocks.

**Pull request merged**